mulheres-de-produto/website
# Adding config to enable progressive migration to the new infrastructure

To enable an incremental migration from the old environment to the new one on Vercel, we need to proxy the pages we still want to maintain to the old website environment.

To see more about the config, please check [here](https://nextjs.org/docs/migrating/incremental-adoption)

To test:
- Access the preview link below
- You can navigate and should see the same content of https://mulheresdeproduto.com/ 
- Access the <preview link>/newhome and you'll see the new home (still in wip) hosted on Vercel environment.
